### PR TITLE
feat: profession view fixes

### PR DESCRIPTION
**Pull request by dubbelnisse**
**Ticket:** [#157](https://trello.com/c/xn14iS8t/157-fixar-p%C3%A5-yrkes-vyn)
**Intent:** Presentation fix when chosen profession, cant go next of not chosen profession and remove profession button.
**How to test (optional):**

### All of the following commands should pass.

- [x] `npm test`
- [x] `npm run lint`
- [x] `npm run cypress`

### Browser testing

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] IE11
- [ ] Edge

### CODE REVIEW

File: src/graphql/resolvers/mutations/removeOccupation.ts [hunk 1]
```diff
@@ -2,20 +2,17 @@ import { InMemoryCache } from 'apollo-cache-inmemory'
...
   _: any,
-  args: any,
+  _args: any,
   { cache }: { cache: InMemoryCache }
-): boolean => {
+): any => {
   cache.writeQuery({
     query: GET_OCCUPATION_CLIENT,
-    data: null,
+    data: { occupation: null },
```
Comment (lundeeell) [inline src/graphql/resolvers/mutations/removeOccupation.ts]: I don't see why this is needed?
If you look in the API, all mutations return boolean (maybe should just return a 204, but however), we should make our client-resolvers as similar to the API-resolvers as possible

File: src/views/CreateProfile/__tests__/ChooseProfession.spec.tsx [hunk 2]
```diff
@@ -35,27 +35,26 @@ describe('views/ChooseProfession', () => {
     ]
   })
 
-  it('renders empty result', async () => {
+  xit('renders empty result', async () => {
```
Comment (lundeeell) [inline src/views/CreateProfile/__tests__/ChooseProfession.spec.tsx]: Maybe remove tests completely

**Comment by dubbelnisse** [comment]
> Clear input button isn't looking that good on phone
> image
> 
> Also, after selecting an occupation, it still looks like an input, I tried several times to "click" the input to edit, before I noticed that you had to clear the field by pressing that clear button. Could that be made more intuitive, like removing the input field border?

**Pull request merged**